src/app/checkwizardpage.cpp

```diff
@@ -77,11 +81,11 @@ bool CheckWizardPage::isComplete() const
 
 int CheckWizardPage::nextId() const
 {
-    if (ui->runRepairRadioButton->isChecked())
+    if (diagnosticTool->anyErrorsInChecks())
     {
         return ADTWizard::Repair_Page;
     }
-    if (ui->finishRadioButton->isChecked())
+    else
```

Comment:
If there are errors user still need to have option to select if he/she run checks.

Reply:
fixed.

Reply:
You did not fix problem that I have described.

Reply:
corrected at this time